distributed/http/templates/worker.html

```diff
@@ -16,7 +16,7 @@ <h2 class="subtitle"> In Memory </h2>
             </tr>
             {% for ts in ws.has_what %}
             <tr>
-              <td> <a href="../task/{{ url_escape(ts.key) }}.html">{{ts.key}}</a></td>
+              <td> <a href="../task/{{ url_escape(str(ts.key)) }}.html">{{ts.key}}</a></td>
```

Comment:
The links to the tasks appear to be broken now.

Reply:
fixed. added tests for all routes